pkg/features/kube_features.go

```diff
@@ -990,7 +997,9 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	LogarithmicScaleDown: {Default: true, PreRelease: featuregate.Beta},
 
-	MatchLabelKeysInPodTopologySpread: {Default: true, PreRelease: featuregate.Beta},
+	MatchLabelKeysInPodAffinity: {Default: false, PreRelease: featuregate.Alpha},
+
+	MatchLabelKeysInPodTopologySpread: {Default: false, PreRelease: featuregate.Alpha},
```

Comment:
it looks like you are ungraduating this by mistake :)

Reply:
Ooops. 🤦 mistaken when rebasing